library/build.gradle

```diff
@@ -3,8 +3,8 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.7.7'
-        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.1"
+        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.3'
```

Comment:
Are the updated libraries here required?

Reply:
fixed https://github.com/Grover-c13/PokeGOAPI-Java/pull/945/commits/c2e6fcd3d851ed025c98a184060469c675a1dd7d

Reply:
It wasn't a request to downgrade -- it was a question of why an update is used here, and is it in scope of the PR?